Darwin-op expose backlash (#2881)
cyberbotics/webots: docs/guide/robotis-op2.md

For more information on the use of all of these sensors/actuators refer to the [Reference Manual](http://www.cyberbotics.com/reference) of Webots.

> *Self Collision*

Reviewer: ```suggestion
#### Self Collision
```

The physical model is very realistic and self collision check is available.
To activate the self collision expand the ROBOTIS OP2 node in the scene tree and set `selfCollision` field to `true`.
Use the self collision check only if you need it, because it is an expensive computation feature and can therefore significantly slow down the simulation speed.

> *Backlash*

Reviewer: ```suggestion
#### Backlash
```

It is possible to model the effect of backlash present in the motors by setting the `backlash` field to `true`.
Doing so will add a backlash of 0.01 [rad] to every joint in the robot.
The modeling of this effect is achieved by replacing the existing joints with [HingeJointWithBacklash](hinge-joint-with-backlash.md) which effectively doubles their number.

> Additionally, for the correct enforcing of this effect, the value of the `basicTimestep` in [WorldInfo](../reference/worldinfo.md) should be set to 8 or lower.

Reviewer: ```suggestion
Additionally, for the correct enforcing of this effect, the value of the `basicTimestep` in [WorldInfo](../reference/worldinfo.md) should probably be set to a smaller value when enabling the backlash.
```

> For these reasons, activating this option is very computationally expensive and will significantly slow down the simulation speed.

Reviewer: ```suggestion
For these reasons, activating this option is computationally expensive and will significantly slow down the simulation speed.
```

export DISPLAY=:0 ./my-controller
```

#### Activating the Backlash Flag Renders the Robot Unable to Stand

> Due to the small amount of backlash present in the motors, for correct modeling of this effect it is required to lower the `basicTimestep` in [WorldInfo](../reference/worldinfo.md) to 8 or less.

Reviewer: ```suggestion
Due to the small amount of backlash present in the motors, for correct modeling of this effect it is required to lower the `basicTimestep` in [WorldInfo](../reference/worldinfo.md). It was observed that setting the `basicTimeStep` to 8 ms gives good results.
```
